pkg/client/client.go

```diff
@@ -105,7 +104,6 @@ func GetGroupKindVersions(group, kind string, versions ...string) (*schema.Group
 	discoveryClient := GetDiscoveryClient()
 	apiGroupResources, err := restmapper.GetAPIGroupResources(discoveryClient)
 	if err != nil {
-		klog.Errorf("could not obtain API group resources: %v", err)
```

Comment:
can we not remove this log line and set it to some `klog.V(5)` instead?

Reply:
We need this log line. It needs to be klog.Errorf only

Reply:
We call this utility function while initializing adapter clients. And this error is expected for fresh installations (before any CRD resources got created). We don't want this log line printed always for fresh installations.

Reply:
```klog.V(3)```